docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -713,13 +713,67 @@ Your new custom cloud is ready to use only after its status changes to
 <TabItem value="2" label="GCP">
 
 1. Generate an IaC template by running [avn byoc create](/docs/tools/cli/byoc#avn-byoc-create).
+
+   ```bash
+   avn byoc create                               /
```

Comment:
The continuation of a line in shell is `\` backslash at the end of the line, not `/` slash.